src/firstboot/firstboot.c

```diff
@@ -1161,6 +1163,76 @@ static int process_root_account(int rfd) {
         return 0;
 }
 
+static int process_regular_accounts(int rfd) {
+        _cleanup_close_ int creds_dfd = -EBADF, dest_dfd = -EBADF;
+        _cleanup_closedir_ DIR *creds_dir = NULL;
+        int r;
+
+        creds_dfd = open_credentials_dir();
```

Comment:
Please follow the logic of `pick_up_credentials`. E.g. some errors here shall be ignored.